test/unit/tsconfig-paths-array-extends/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+	"extends": ["./tsconfig.a.json", "./tsconfig.b.json"],
```

Comment:
> Path to base configuration file to inherit from (requires TypeScript version 2.1 or later), or array of base files, with the rightmost files having the greater priority (requires TypeScript version 5.0 or later).

That is, the priorities are:

1. `tsconfig.json`
2. `tsconfig.b.json`
3. `tsconfig.a.json`

This test verifies that the override of the `allowJs` property and the combination of A and B are working as expected.